prboom2/src/am_map.c

```diff
@@ -1218,6 +1278,9 @@ void AM_Ticker (void)
 
   if (stop_zooming && leveltime - zoom_leveltime != 1)
     AM_StopZooming();
+
+  if (dsda_IntConfig(dsda_config_map_trail))
+    AM_updateTrail();
```

Comment:
I think this isn't quite exactly where we want this to happen. The critical action is when the player's position is updated, especially if it means they can trigger lines at that moment. That happens at the tail end of P_TryMove. It's not quite the same as a once per tick position check because high speeds cause multiple P_TryMove calls within a tic (this is why wallrunning works, for example).

Reply:
Yup. I thought about split movements and moved it to the back of my mind, as something to tackle later, and then completely forgot about it.